Miniforge3/construct.yaml

```diff
@@ -1,7 +1,7 @@
 {% set name = os.environ.get("MINIFORGE_NAME", "Miniforge3") %}
 {% set version = os.environ.get("MINIFORGE_VERSION", "24.7.1-0") %}
 {% set conda_libmamba_solver_version = "24.7.0"%}
-{% set mamba_version = "1.5.8"%}
+{% set mamba_version = "1.5.9"%}
```

Comment:
There is also the     MICROMAMBA_VERSION=1.5.8 in https://github.com/jaimergp/miniforge/blob/system-requirements-and-conda-24.7.1/scripts/build.sh#L41
If I recall correctly they are usually upgraded together

Reply:
thanks its often forgotten.

Reply:
New micromamba requires the newer image for a more recent GLIBC, it looks like, so I bumped the image for x64 which was still using 2.12 IIRC.